src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -1310,6 +1370,9 @@ export default class ResultsViewOncoprint extends React.Component<
                     ),
                     ...this.genericAssayHeatmapTracks.result.map(
                         hmTrack => hmTrack.trackGroupIndex
+                    ),
+                    ...this.genericAssayCategoricalTracks.result.map(
+                        track => track.trackGroupIndex
```

Comment:
Do we want to add `CategoricalTracks` in `HeatmapTrackGroup`? Not sure if this is because we are using `HeatmapTrack` to represent `AdditionalTrack`? If so, maybe we should rename some of the variables.

Reply:
this is just computing what group index will be the geneset heatmaps. this just mean sit will go all the way at the end after heatmap tracks, generic assay heatmap tracks, and generic assay categorical tracks